src/dataset.py

```diff
@@ -220,10 +220,10 @@ def build_labels(self, data):
 
         for r, (row, token_row)  in enumerate(zip(data[0], self.tokens)):  # get each row of data
             # add placeholder for [CLS] token
-            row_expression = [0]  # similar to RACL, use padding token
-            row_holder = [0]  # similar to RACL, use padding token
-            row_polarity = [0]  # similar to RACL, use padding token
-            row_target = [0]  # similar to RACL, use padding token
+            row_expression = [self.IGNORE_ID]  # ignore this token during evaluation
+            row_holder = [self.IGNORE_ID]  # ignore this token during evaluation
+            row_polarity = [self.IGNORE_ID]  # ignore this token during evaluation
+            row_target = [self.IGNORE_ID]  # ignore this token during evaluation
```

Comment:
Now ignoring CLS and SEP from predictions, instead of padding like RACL did. This should help metrics slightly (at least precision may see an increase)